docs/analytics-gtm/gtm-link-click-tracker.md

```diff
@@ -0,0 +1,62 @@
+# Google Tag Manager link click tracker
+
+A module exists for tracking link clicks on GOV.UK.
+
+When initialised, the JS adds an event listener to the `<body>` of the page with the following listeners:
+
+- A `click` listener (for left clicks, control clicks, and "Meta key" clicks - "Meta key" clicks are command + clicks or windows key + clicks.)
```

Comment:
Genuine interest - what's a windows key + click? I thought if you press the windows key it opens the Start menu?

Reply:
@andysellick `command` and `windows key` are considered meta keys. If you pressed the windows key alone it would open the start menu, but not if you  held it and clicked at the same time. It looks like Windows key + Click doesn't do anything different on links, but on a Mac the meta key + click opens the link in a new tab. 

I doubt anyone on a Windows machine will do a Windows Key + Click so I don't think we will need to check in the JS wether they are on a Mac when tracking this. But if you think we should ensure they are on a Mac first, then I can add an if statement to check which platform the user is on. 

I believe you can filter GA reports by Operating System as well, so if we were getting Windows Key + Click events they could be filtered out to only show Command + Click events.

Reply:
No that's fine I was just interested, thanks for the info.